src/transform/namer.ts

```diff
@@ -51,6 +51,10 @@ export async function namer(session: Session<CodeModel>) {
   // copy all the .language.default data into .language.go
   cloneLanguageInfo(model);
 
+  // default namespce to the output folder
+  const outputFolder = await session.getValue<string>('output-folder');
+  model.language.go!.packageName = outputFolder.substr(outputFolder.lastIndexOf('/') + 1);
```

Comment:
So the namespace will always be defined by the last directory in the output-folder path?

Reply:
Correct, the two have to match (this is how it was, I just moved it to here).